src/Components/CollectionsHubRails/ArtistSeriesRail/__tests__/ArtistSeriesEntity.test.tsx

```diff
@@ -13,34 +13,86 @@ describe("ArtistSeriesEntity", () => {
     }
   })
 
-  it("Renders expected fields", () => {
+  it("showing the correct text, price guidance, amount of hits and image", () => {
     const component = mount(<ArtistSeriesEntity {...props} />)
-
-    expect(component.text()).not.toMatch("Jasper Johns:")
-    expect(component.text()).toMatch("Flag")
+    expect(component.text()).toMatch("Flags unique collections")
     expect(component.text()).toMatch("From $1,000")
     expect(component.find(ArtworkImage).length).toBe(3)
     const artworkImage = component
       .find(ArtworkImage)
       .at(0)
       .getElement().props
-
     expect(artworkImage.src).toBe(
       "https://d32dm0rphc51dk.cloudfront.net/4izTOpDv-ew-g1RFXeREcQ/small.jpg"
     )
-    expect(artworkImage.alt).toBe("Jasper Johns, Flag")
+  })
+
+  it("NOT rendering any artist name or the title of the artwork ", () => {
+    const component = mount(<ArtistSeriesEntity {...props} />)
+    expect(component.text()).not.toMatch("A great flag from Jasper")
+    expect(component.text()).not.toMatch("Jasper Johns")
+    expect(component.text()).not.toMatch("Back to 2046")
+    expect(component.text()).not.toMatch("Andy Warhol")
+    expect(component.text()).not.toMatch("An Apple")
+  })
+
+  it("uses small image width when there are more than 2 hits", () => {
+    const component = mount(<ArtistSeriesEntity {...props} />)
+    const artworkImage = component
+      .find(ArtworkImage)
+      .at(0)
+      .getElement().props
+    expect(component.find(ArtworkImage).length).toBe(3)
     expect(artworkImage.width).toBe(85)
   })
 
-  it("Returns proper image size if 2 artworks returned", () => {
+  it("uses medium image width when there are only 2 hits", () => {
     props.member.artworks.hits.pop()
     const component = mount(<ArtistSeriesEntity {...props} />)
     const artworkImage = component
       .find(ArtworkImage)
       .at(0)
       .getElement().props
-
     expect(component.find(ArtworkImage).length).toBe(2)
     expect(artworkImage.width).toBe(131)
   })
+
+  it("uses large image width when there are only 1 hits", () => {
+    props.member.artworks.hits.pop()
+    const component = mount(<ArtistSeriesEntity {...props} />)
+    const artworkImage = component
+      .find(ArtworkImage)
+      .at(0)
+      .getElement().props
+    expect(component.find(ArtworkImage).length).toBe(1)
+    expect(artworkImage.width).toBe(265)
+  })
+
+  it("uses the hit title for alt text if there is no artist", () => {
+    const component = mount(<ArtistSeriesEntity {...props} />)
+    const artworkImage = component
```

Comment:
See my above comment about moving the more general assertion about the number of `ArtworkImage` elements above this line.